pkgs/top-level/all-packages.nix

```diff
@@ -9817,9 +9817,8 @@ with pkgs;
   inherit (callPackage ../servers/plik { })
     plik plikd;
 
-  plex = callPackage ../servers/plex {
-    buildFHSUserEnv = buildFHSUserEnvBubblewrap;
```

Comment:
@bachp Do you happen to recall why you said that this commit switched Plex to use Bubblewrap, when this line is evidence that it was using Bubblewrap before it?

Reply:
@winterqt I think this was the result of rebasing on top #132963